tests/screenshot.test.js

```diff
@@ -0,0 +1,23 @@
+
+const path = require('path');
+const timeout = 5000
+
+describe(
+    '/ (ostrich)',
+    () => {
+        let page
```

Comment:
There seem to be quite a lot of code-duplication here w.r.t. the other test-case. How can the equal parts be extracted into some re-usable component?